### PR TITLE
Fix spacing/linking of s3 backup section

### PR DESCRIPTION
**Pull request by benjaminjb**
**Checklist:**
 - [ ] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [x] Have you updated or added documentation for the change, as applicable?
 - [ ] Have you tested your changes on all related environments with successful results, as applicable?
   - [ ] Have you added automated tests?

**Type of Changes:**
 - [ ] New feature
 - [ ] Bug fix
 - [x] Documentation
 - [ ] Testing enhancement
 - [ ] Other

**What is the current behavior (link to any open issues here)?**

Spacing/linking error

**What is the new behavior (if this is a feature change)?**
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

Fixed error in one small section.

**Other Information**:

### CODE REVIEW

File: docs/content/tutorial/backups.md [hunk 1]
```diff
@@ -137,10 +137,10 @@ variables will then be used by pgBackRest to assume the identity of a role that
 to upload to an S3 repository.
 
 This method requires [additional setup in AWS IAM](https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.html).
-Follow the procedure there, but skip the third step:
+Follow the procedure there for steps one and two:
```
Comment (jkatz) [inline docs/content/tutorial/backups.md:140]:
```suggestion
Use the procedure in the linked documentation for the first two steps described below:
```

File: docs/content/tutorial/backups.md [hunk 2]
```diff
@@ -137,10 +137,10 @@ variables will then be used by pgBackRest to assume the identity of a role that
...
 
-1\. Create an OIDC provider for your EKS cluster
-2\. Create an IAM policy for bucket access and an IAM role with a trust relationship with the 
+1. Create an OIDC provider for your EKS cluster
```
Comment (jkatz) [inline docs/content/tutorial/backups.md:142]:
```suggestion
1. Create an OIDC provider for your EKS cluster.
```

**Review by cbandy — APPROVED** [review]

**Review by jkatz — APPROVED** [review]

File: docs/content/tutorial/backups.md [hunk 3]
```diff
@@ -150,7 +150,7 @@ You can then make the following changes to the files in the `kustomize/s3` direc
 [Postgres Operator examples](https://github.com/CrunchyData/postgres-operator-examples/fork) repository:
 
 1\. Add the `s3` section to the spec in `kustomize/s3/postgres.yaml` as discussed in the 
-`Using S3 Credentials` section. In addition to that, add the required `eks.amazonaws.com/role-arn` 
+`Using S3 Credentials` [section above](#using-s3-credentials). In addition to that, add the required `eks.amazonaws.com/role-arn`
```
Comment (cbandy) [inline docs/content/tutorial/backups.md:153]: 🤔 I interpret the monospace formatting of backticks as code that should be taken literally. Having it on "Using S3 Credentials" is not what I would expect. It's literal, quotes of some kind make sense, so it's not confusing exactly.

🤔 A link on those words makes more sense to me, but the basis is entirely subjective. Do quotes belong around something like this? 🤷🏻 I'd love for us to use technical writing guidelines for things like this.

```suggestion
[Using S3 Credentials](#using-s3-credentials) section above. In addition to that, add the required `eks.amazonaws.com/role-arn` 
```

Comment (benjaminjb) [reply docs/content/tutorial/backups.md:153]: Whoops, sorry about that, missed this when merging, hold on.

**Pull request merged**